doc/source/r_interface.rst

```diff
@@ -60,7 +60,7 @@ can be replaced with::
 
     from rpy2.robjects import r
     r.data('iris')
```

Comment:
sure if that works, just make a note (e.g. import at the top and indicate), you are importing an external library to do this.